src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -226,14 +243,16 @@ private static bool Compile(ProjectContext context, string configuration, string
                 foreach (var projectDependency in Sort(projects))
                 {
                     // Skip compiling project dependencies since we've already figured out the build order
-                    var compileResult = Command.Create("dotnet-compile", 
+                    var compileResult = ResolveCommand("dotnet-compile", 
```

Comment:
What do you think of leaving this as Command.Create (more readable IMO), do some checks to determine the safety, then call execute?